### ISSUE TITLE
can't install solo5-bindings-hvt/spt on fedora

### ISSUE THREAD

**Issue by anuragsoni** [Solo5/solo5#458]
Hello,

I'm trying build a mirageOS unikernel targetting the hvt (i also tried this with spt) backend, and i'm getting a compilation error when opam tries to install solo5-bindings-hvt. My OS is Fedora 32

```
[ERROR] The compilation of solo5-bindings-hvt failed at "/home/asoni/.opam/opam-init/hooks/sandbox.sh build ./configure.sh".

#=== ERROR while compiling solo5-bindings-hvt.0.6.4 ===========================#
# context     2.0.6 | linux/x86_64 | ocaml-variants.4.10.0+flambda | https://opam.ocaml.org#88a179f5
# path        ~/.opam/4.10.0+flambda/.opam-switch/build/solo5-bindings-hvt.0.6.4
# command     ~/.opam/opam-init/hooks/sandbox.sh build ./configure.sh
# exit-code   1
# env-file    ~/.opam/log/solo5-bindings-hvt-164580-d2c37b.env
# output-file ~/.opam/log/solo5-bindings-hvt-164580-d2c37b.out
### output ###
# configure.sh: ERROR: Only 'gcc' 4.x+ is supported on Linux
```

I have gcc installed, and i've installed all the dependencies that depext could find. (kernel-headers is installed). 

OS: Fedora 32
gcc: 10.0.1
mirage: v3.7.6

**Comment by anuragsoni** [comment]
https://github.com/Solo5/solo5/blob/26bc7f2eae311e18f3f1eae5caf5c8f322f7ca8c/configure.sh#L35
This is probably the culprit for the build failure

**Comment by mato** [comment]
Fixed by #459.
